index.html

```diff
@@ -7,57 +7,215 @@
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+            </section>
+            <!-- about section -->
+            <section class="about">
+                <h2>About me</h2>
```

Comment:
- [ ] Kindly ensure that you follow suit with all the design guidelines to achieve an optimized pixel-perfect design like you'll be doing in your next job. Note that everything must much as possible, the font size, font weight, margin, padding, background image, color, line height, font family, and the like.

- Your design
<img width="163" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227706092-07d669d3-4716-4a10-9fe6-8336314ad7cc.png">

- Figma design
<img width="291" alt="right" src="https://user-images.githubusercontent.com/41163571/227706103-b1041498-b057-4a03-bf36-4be057ddda22.png">

I took the extra mile for your sake to confirm if the font existed and was for free, guess what? 
[it is!!!!!!!](https://fonts.google.com/specimen/Crete+Round?query=crete).

- You can always get the properties on Figma by clicking on the item, and then check the right side pane for the information.

Reply:
Incorporated the font as suggested.